CONTRIBUTING.md

```diff
@@ -71,9 +74,9 @@ test can erode this trust.
   - If the test is failing consistently it might be a sign that something
     different in the stack is broken
 - Next we want to stop the test failing for users of the suite:
-  - Create a PR which skips the flaky test in this repository e.g.
+  - Create a PR which marks the flaky test as flaky in this repository e.g.
```

Comment:
I would remove the first 'flaky' here, so it reads "marks the test as flaky".

Reply:
Ah we updated this locally but forgot to save it :man_facepalming: